docs/changelogs/0.8.1.rst

```diff
@@ -0,0 +1,44 @@
+=====
+0.8.1
+=====
+
+**Release date:** 2020-11-24
+
+Breaking changes
+================
+
+None
+
+New features
+============
+
+Updated list filtering capabilities of the Django adapter
+---------------------------------------------------------
+
+More policies and Polar operations can now be used in the context of list
+filtering. In particular, the ``in`` and ``matches`` operators (including
+specializers in rule heads and embedded fields) now work in more cases,
+as do nested lookups and comparison operators. The list filtering feature
+remains in preview; we would love to hear feedback if you're trying to use
+it in your application.
+
+Install the latest version from pip: ``pip install django-oso``.
+
+Check out `the documentation <TODO>`_.
+
+SQLAlchemy adapter for list filtering
+-------------------------------------
+
+We've added our second list filtering adapter: SQLAlchemy! It is
```

Comment:
This is way broader than just list filtering, right? This is our sqlalchemy integration!

Reply:
You are right, thanks. Got stuck in the list filtering mindset.